README.md

```diff
@@ -154,6 +154,7 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 * [LynxKite](https://lynxkite.com/) - A complete graph data science platform for very large graphs and other datasets.
 * [ML Workspace](https://github.com/ml-tooling/ml-workspace) - All-in-one web-based IDE specialized for machine learning and data science.
 * [Modzy](https://www.modzy.com/) - AI platform and marketplace offering scalable, secure, and ready-to-deploy AI models.
+* [Neu.ro](https://neu.ro) - MLOps platform that orchestrates resources, processes, and access, and integrates specific open-source and proprietary MLOps tools into client-oriented MLOps systems.
```

Comment:
Can you please make the description shorter to fit the Github UI avoiding the line break?

For example:

```
Integrates specific open-source and proprietary tools into client-oriented MLOps systems.
```

Reply:
@kelvins how about new version?

Reply:
Excellent, thanks for the contribution :tada: